packages/components/stories/Toggle.js

```diff
@@ -58,4 +77,26 @@ storiesOf('Toggle', module)
 				<Toggle {...withLabel} />
 			</form>
 		</div>
+	))
+	.add('Label toggle', () => (
+		<div>
+			<h1>Label Toggle</h1>
+			<form>
+				<h3>Non interactive 2 state</h3>
+				<Toggle.Label
```

Comment:
[onChange](https://github.com/Talend/ui/pull/2633/files#diff-ae7c2fecdebbf4d67bf8cb1e46fdf472R8) is missing for each of these new stories